app/mutations/Manager/addManagerToRevision.ts

```diff
@@ -0,0 +1,54 @@
+import { useMutation, graphql, Disposable, Environment } from "react-relay";
+import { MutationConfig } from "relay-runtime";
+import { addManagerToRevisionMutation } from "__generated__/addManagerToRevisionMutation.graphql";
+
+export const mutation = graphql`
+  mutation addManagerToRevisionMutation(
+    $projectRevision: ID!
+    $projectRevisionId: Int!
+    $newFormData: JSON!
+  ) {
+    createFormChange(
+      input: {
+        formDataSchemaName: "cif"
+        formDataTableName: "project_manager"
+        jsonSchemaName: "project_manager"
+        operation: CREATE
+        changeReason: "Add manager to project revision from project manager form"
+        projectRevisionId: $projectRevisionId
+        newFormData: $newFormData
+      }
+    ) {
+      query {
+        projectRevision(id: $projectRevision) {
+          managerFormChanges: projectManagerFormChangesByLabel {
+            edges {
+              node {
+                projectManagerLabel {
+                  id
+                  rowId
+                  label
+                }
+                formChange {
+                  id
+                  newFormData
+                }
+              }
+            }
```

Comment:
https://bcgov-cas.promyze.app/#/CAS%20GGIRCS/home?practiceId=61e1eb156e4cd70fe286ec9f
You could reuse `ProjectManagerForm_query` here, or, if that fragment was split in two, `ProjectManagerForm_projectRevision`